### ISSUE TITLE
x_to_nwb.ABFConverter failed for clamp mode 2

### ISSUE THREAD

**Issue by bendichter** [AllenInstitute/ipfx#319]
`ABFConverter(src_path, dest_path, outputFeedbackChannel=True)`

```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-3-73dd97f2f1b8> in <module>
      7     dest_path = '/'.join(src_path.split('/')[:6] + ['nwb', dest_fname])
      8 
----> 9     ABFConverter(src_path, dest_path, outputFeedbackChannel=True)
     10 
     11 

~/dev/ipfx/ipfx/x_to_nwb/ABFConverter.py in __init__(self, inFileOrFolder, outFile, outputFeedbackChannel, compression)
     79         nwbFile.add_ic_electrode(electrodes)
     80 
---> 81         for i in self._createStimulusSeries(electrodes):
     82             nwbFile.add_stimulus(i)
     83 

~/dev/ipfx/ipfx/x_to_nwb/ABFConverter.py in _createStimulusSeries(self, electrodes)
    391                                              sort_keys=True, indent=4)
    392 
--> 393                     seriesClass = getStimulusSeriesClass(self._getClampMode(abf, channel))
    394 
    395                     stimulus = seriesClass(name=name,

~/dev/ipfx/ipfx/x_to_nwb/conversion_utils.py in getStimulusSeriesClass(clampMode)
     54         return CurrentClampStimulusSeries
     55     else:
---> 56         raise ValueError(f"Unsupported clamp mode {clampMode}.")
     57 
     58 

ValueError: Unsupported clamp mode 2.
```

Looking at 
https://github.com/AllenInstitute/ipfx/blob/050ed97c9590996e099c9e4f7609b62bd7847e8a/ipfx/x_to_nwb/conversion_utils.py#L22-L23

it looks like only modes 0 and 1 are supported, corresponding to voltage and current clamps. What is mode 2? Can we add support for it?

I can see about sharing the data if that is necessary. It's not mine so I will need to get permission first.

fyi @satra, @t-b

**Comment by t-b** [comment]
I'm guessing 2 is `I=0`, it is at least for Multiclamp 700B amplifiers. Having a sample file would be good.

**Comment by bendichter** [comment]
Yes, this was recorded using Multiclamp 700B amplifiers and you are right, this is a zero-current sweep.

**Comment by t-b** [comment]
@bendichter This was resolved by #346. Please reopen if not.
